config/jmxtrans.xml

```diff
@@ -0,0 +1,144 @@
+<jmxtrans-agent>
+  <queries>
```

Comment:
add a document for worker startup parameter.
1. describe jmx parameter
2. describe 1-1 mapping parameter